requirements-test.txt

```diff
@@ -1,2 +1,4 @@
 pytest
-pytest-cov
\ No newline at end of file
+pytest-cov
+ml_wrappers
+vision_explanation_methods
```

Comment:
this is the vision-explanation-methods package - it is already installed from: 
```
pip install -e .
```
hence, this line can just be removed